elasticjob-ecosystem/elasticjob-tracing/elasticjob-tracing-rdb/src/main/java/org/apache/shardingsphere/elasticjob/tracing/rdb/storage/RDBJobEventStorage.java

```diff
@@ -61,14 +63,41 @@ public final class RDBJobEventStorage {
     private final DatabaseType databaseType;
     
     private final RDBStorageSQLMapper sqlMapper;
+
+    private static final Map<DataSource, RDBJobEventStorage> STORAGE_MAP = new ConcurrentHashMap<>();
+
     
     static {
         for (DatabaseType each : ServiceLoader.load(DatabaseType.class)) {
             DATABASE_TYPES.put(each.getType(), each);
         }
     }
+
+    public static RDBJobEventStorage getInstance(final DataSource dataSource) throws SQLException {
+        return wrapException(() -> STORAGE_MAP.computeIfAbsent(dataSource, (ds) -> {
+            try {
+                return new RDBJobEventStorage(ds);
+            } catch (SQLException e) {
+                throw new RuntimeException(e);
+            }
+        }));
+    }
+
+    public static RDBJobEventStorage wrapException(Callable<RDBJobEventStorage> callable) throws SQLException {
```

Comment:
How about using `Supplier`?

Reply:
I would love to! I've modified it, please check it out

Reply:
![image](https://user-images.githubusercontent.com/46399833/220915814-6d22d016-c86d-4eef-83fc-1af8c8749eb3.png)
https://github.com/apache/shardingsphere-elasticjob/actions/runs/4248881615/jobs/7388688271
 
The above link is the information about the failure of the last build. From the error information, we can see that zookeeper was closed for some reason, which caused the unit test to fail. I tried executing the command locally with success: 

```
./mvnw --batch-mode --no-transfer-progress '-Dmaven.javadoc.skip=true' clean install
```
I  can not reproduce the above error, what do you suggest?